### ISSUE TITLE
[Bug] java.lang.Throwable: Expected to set [java.awt.Color[r=0,g=0,b=0],null,0,BOXED,null,{},null] but actual attributes are: [java.awt.Color[r=0,g=0,b=128],null,0,BOXED,null,{},null]; colorsScheme: '_@user_IntelliJ Light'; highlighter:RangeHighlighter: (1652,1656); layer:3000; tooltip: HighlightInfo(1652,1656) text='type'; severity=INFORMATION; group=4 (class com.intellij.openapi.editor.impl.RangeHighlighterImpl); markup: MarkupModel for DocumentImpl[VirtualFile: C:\Users\Mohammad Salehpour\PycharmProjects\DS_ML\Session+03+-+Matplotlib+&+Searbon(Visualization).ipynb] (class com.intellij.openapi.editor.impl.MarkupModelImpl); attributes after the second .setAttributes(): [java.awt.Color[r=0,g=0,b=0],null,0,BOXED,null,{},null]

### ISSUE THREAD

**Issue by Mohammad-Salehpour** [dinbtechit/vscode-theme#121]
#Describe the bug

#To Reproduce

#Expected behavior

#Screenshots

#Environment
* Java: JetBrains s.r.o. 17.0.7
* OS: Windows 11 amd64
* IDE: PyCharm 2023.2
* Version: 1.10.1

#Additional context

#Stacktrace#
 
```
java.lang.Throwable: Expected to set [java.awt.Color[r=0,g=0,b=0],null,0,BOXED,null,{},null] but actual attributes are: [java.awt.Color[r=0,g=0,b=128],null,0,BOXED,null,{},null]; colorsScheme: '_@user_IntelliJ Light'; highlighter:RangeHighlighter: (1652,1656); layer:3000; tooltip: HighlightInfo(1652,1656) text='type'; severity=INFORMATION; group=4 (class com.intellij.openapi.editor.impl.RangeHighlighterImpl); markup: MarkupModel for DocumentImpl[VirtualFile: C:\Users\Mohammad Salehpour\PycharmProjects\DS_ML\Session+03+-+Matplotlib+&+Searbon(Visualization).ipynb] (class com.intellij.openapi.editor.impl.MarkupModelImpl); attributes after the second .setAttributes(): [java.awt.Color[r=0,g=0,b=0],null,0,BOXED,null,{},null]
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:370)
	at com.intellij.codeInsight.daemon.impl.BackgroundUpdateHighlightersUtil.createOrReuseHighlighterFor(BackgroundUpdateHighlightersUtil.java:309)
	at com.intellij.codeInsight.daemon.impl.BackgroundUpdateHighlightersUtil.addHighlighterToEditorIncrementally(BackgroundUpdateHighlightersUtil.java:74)
	at com.intellij.codeInsight.daemon.impl.HighlightingSessionImpl.addInfoIncrementally(HighlightingSessionImpl.java:196)
	at com.intellij.codeInsight.daemon.impl.DefaultHighlightInfoProcessor.infoIsAvailable(DefaultHighlightInfoProcessor.java:159)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.queueInfoToUpdateIncrementally(GeneralHighlightingPass.java:492)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass$1.add(GeneralHighlightingPass.java:481)
	at com.intellij.codeInsight.daemon.impl.DefaultHighlightVisitor$1.queueToUpdateIncrementally(DefaultHighlightVisitor.java:85)
	at com.intellij.codeInsight.daemon.impl.B.create(B.java:318)
	at com.github.dinbtechit.vscodetheme.annotators.BaseAnnotator.annotate(BaseAnnotator.kt:26)
	at com.intellij.codeInsight.daemon.impl.DefaultHighlightVisitor.runAnnotators(DefaultHighlightVisitor.java:130)
	at com.intellij.codeInsight.daemon.impl.DefaultHighlightVisitor.visit(DefaultHighlightVisitor.java:107)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.runVisitors(GeneralHighlightingPass.java:360)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.lambda$collectHighlights$7(GeneralHighlightingPass.java:291)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.analyzeByVisitors(GeneralHighlightingPass.java:320)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.lambda$analyzeByVisitors$8(GeneralHighlightingPass.java:323)
	at com.intellij.codeInsight.daemon.impl.DefaultHighlightVisitor.analyze(DefaultHighlightVisitor.java:92)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.analyzeByVisitors(GeneralHighlightingPass.java:323)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.collectHighlights(GeneralHighlightingPass.java:286)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.collectInformationWithProgress(GeneralHighlightingPass.java:234)
	at com.intellij.codeInsight.daemon.impl.ProgressableTextEditorHighlightingPass.doCollectInformation(ProgressableTextEditorHighlightingPass.java:95)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:57)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$1(PassExecutorService.java:390)
	at com.intellij.platform.diagnostic.telemetry.helpers.TraceKt.runWithSpanIgnoreThrows(trace.kt:77)
	at com.intellij.platform.diagnostic.telemetry.helpers.TraceUtil.runWithSpanThrows(TraceUtil.java:24)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$2(PassExecutorService.java:386)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1133)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$3(PassExecutorService.java:377)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$13(CoreProgressManager.java:604)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:679)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:635)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:603)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:61)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.doRun(PassExecutorService.java:376)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$run$0(PassExecutorService.java:352)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:201)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:207)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:350)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:185)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
```

**Comment by dinbtechit** [comment]
@Mohammad-Salehpour  - Thank you for reporting this error. Are you able to provide some screenshots when this error occurs?

**Comment by dinbtechit** [comment]
closing this this will be addressed in #123
